chapters/8-annotations.md

```diff
@@ -114,15 +114,19 @@ Example:
 
 **8.4.6** RDF:
 
-For RDF, the annotations are a property of the SPDX element it is annotationg.
-
-    <SpdxElement rdf:about=”#SPDXRef-45”>
-        <annotation>
-            <Annotation>
-                ...
-            </Annotation>
-        </annotation>
-    </SpdxElement rdf:about=”#SPDXRef-45”>
+For RDF, the annotations may be a property of the SPDX element it is annotating.
+
+    <SpdxElement rdf:about="#SPDXRef-45">
+        <Annotation>
+            ...
+        </Annotation>
+    </SpdxElement rdf:about="#SPDXRef-45">
+
+Alternatively, they may contain an `rdf:about` attribute with an SPDXID.
+
+    <Annotation rfd:about="#SPDXRef-45">
+        ...
```

Comment:
I believe the `rdf:about` in this case resolves to the URI for the `Annotation`.  The intent is that the `Annotation` is an `annotation` property of the `SpdxElement`.  You could assign a unique ID for the `Annotation` and refer to that ID.  The example would be:
```
<SpdxElement rdf:about="#SPDXRef-45">
        <annotation rdf:about="#AnnotationId-12"/>
         ...
</SpdxElement rdf:about="#SPDXRef-45">
<Annotation rdf:about="#AnnotationId-12>
    <rdfs:comment>Comment</rdfs:comment>
     ...
</Annotation>
```